docs/send-data/opentelemetry-collector/data-source-configurations/mapping-records-resources.md

```diff
@@ -0,0 +1,82 @@
+---
+id: mapping-records-resources
+title: Mapping OpenTelemetry Concepts to Sumo Logic
+sidebar_label: Mapping OTel Concepts to Sumo Logic
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This page describes how to map your OpenTelemetry attributes like Resources, Records, and Instrumentation Scope, to Sumo Logic concepts like Fields.
+
+OpenTelemetry has a [rich data model](https://github.com/open-telemetry/opentelemetry-proto/tree/main/opentelemetry/proto) that is internally constructed out of several layers. For all signals, these can be broken down into following:
+
+* **Resource**. Includes attributes describing the resource from which given set of data comes from. Should follow [resource semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions).
+* **Instrumentation Scope**. Additional information about the scope of data. For example, instrumentation library name.
+* **Record**. Refers to a specific entry of data, such as a Log, Span, or Metric.
+:::note
+For logs, this is usually called a _Record_; for traces, it's usually called a _Span_; and for metrics, a _Datapoint_ (or _sample_, if Prometheus is used). In this documentation, we'll use _Record_ for all of these.
+:::
+Each Record has its own set of attributes, which may include key/value pairs that are specific to the context of the Record. Logs, in particular, can also include attributes in the body of the Record. Some Record types may follow certain conventions for signal types, such as [trace](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/trace/semantic_conventions), [metrics](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions), or [logs](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/logs/semantic_conventions).
```

Comment:
@swiatekm-sumo Have I captured correctly what you meant [here](https://github.com/SumoLogic/sumologic-documentation/pull/3461#discussion_r1441615814)?